web_calendar_custom/static/src/js/web_calendar_custom.js

```diff
@@ -0,0 +1,132 @@
+/*
+    © 2018 Savoir-faire Linux <https://savoirfairelinux.com>
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/LGPL.html).
+*/
+
+// TODO: Use model ir.config_paramater and add above fields
+
+odoo.define('web_calendar_custom.renderer_setting', function (require) {
+"use strict";
+
+    var rpc = require('web.rpc');
+    var AbstractRenderer = require('web.AbstractRenderer');
+    var relational_fields = require('web.relational_fields');
+    var FieldManagerMixin = require('web.FieldManagerMixin');
+    var field_utils = require('web.field_utils');
+    var session = require('web.session');
+    var Dialog = require('web.Dialog');
+    var Widget = require('web.Widget');
+    var utils = require('web.utils');
+    var core = require('web.core');
+    var QWeb = require('web.QWeb');
+
+    var _t = core._t;
+    var qweb = core.qweb;
+
+
+    String.prototype.toHHMMSS = function () {
+        var hour_num = parseFloat(this); // don't forget the second param
+        var hours   = Math.trunc(hour_num);
+        var minutes = (hour_num - hours) * 60;
+
+        if (hours   < 10) {hours   = "0"+hours;}
+        if (minutes < 10) {minutes = "0"+minutes;}
+        return hours+':'+minutes+':'+'00';
+    }
+
+    /* Get configuration values from res.config.settings */
+    rpc.query({
+             model: 'res.config.settings',
+             method: 'get_values',
+             args: ['']
+    }).then(function (returned_values) {
+              /* Override fullCalendar configuration */
+            var Thread = AbstractRenderer.include({
```

Comment:
@lgharib why you didnt extend?

Reply:
We cannot extend here because extending is used for changing a function, however in this case we are reconfiguring the full calender using different options in the options dictionary.